staking_deposit/utils/ssz.py

```diff
@@ -56,6 +57,17 @@ def compute_deposit_domain(fork_version: bytes) -> bytes:
     return domain_type + fork_data_root[:28]
 
 
+def compute_voluntary_exit_domain(fork_version: bytes, genesis_validators_root: bytes) -> bytes:
+    """
+    VOLUNTARY_EXIT-only `compute_domain`
```

Comment:
Provide ref https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/beacon-chain.md#compute_domain

Reply:
Not needed